src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../types';
+import { findPersonByName } from '../utils/findPersonByName';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+
+export const PersonInfo: FC<Props> = ({ person, people }) => {
+  const { personSlug } = useParams();
+
+  const mother = findPersonByName(people, person.motherName);
+  const father = findPersonByName(people, person.fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': person.slug === personSlug })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={cn({
+            'has-text-danger': person.sex === 'f',
+          })}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {mother ? (
+          <Link to={`/people/${mother.slug}`} className={'has-text-danger'}>
+            {person.motherName}
+          </Link>
+        ) : (
+          <>{person.motherName || '-'}</>
+        )}
```

Comment:
You are repeating similar JSX for mother and father. Consider creating a separate component for this to avoid code duplication.